### DISCUSSION TITLE
How to get the sql Lineage analysis and the bdbje monitoring

### DISCUSSION THREAD

**Discussion by zhangweiwhim** [apache/doris#30427] [RFC]
as the title, first of all
1. Lineage analysis that I searched tools like SQLLineage, DataHub, and OpenMeta, etc, do not support  Doris, some projects I searched in GitHub that I tried do not work perfectly,  plz give some Realize the thinking direction.

3. For the bdbje monitoring, I searched a method like below URL, but I do not know that useful
https://docs.oracle.com/cd/E17277_02/html/jconsole/JConsole-plugin.html,
 has any other solution to monitor bdbje.
thx

**Comment by likun1210** [comment]
> as the title, first of all
> 
> 1. Lineage analysis that I searched tools like SQLLineage, DataHub, and OpenMeta, etc, do not support  Doris, some projects I searched in GitHub that I tried do not work perfectly,  plz give some Realize the thinking direction.
> 2. For the bdbje monitoring, I searched a method like below URL, but I do not know that useful
>    https://docs.oracle.com/cd/E17277_02/html/jconsole/JConsole-plugin.html,
>    has any other solution to monitor bdbje.
>    thx

May I ask about Doris' data lineage how did you implement it?
请问doris的数据血缘你后面是怎么实现的？
